packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -423,20 +462,28 @@ public void onPathReady(String path) {
   }
 
   private void handleImageResult(String path, boolean shouldDeleteOriginalIfScaled) {
-    if (pendingResult != null) {
+    if (methodCall != null) {
       Double maxWidth = methodCall.argument("maxWidth");
```

Comment:
Now that we're parsing these in multiple places, would it make sense to parse them once and then store them somewhere instead? Reduces the surface where we could get typing errors here.